.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,48 @@
+name: Feature request
+description: Suggest an idea for this project
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this issue! :hugs:
+        
+        However note the following before opening an issue:
+        * IF YOU DON'T FILL IN THE TEMPLATE PROPERLY, YOUR ISSUE IS LIABLE TO BE CLOSED.
+        * If you feel tired/lazy right now, open your issue some other time. We'll wait.
+
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: "Checklist"
+      options:
+        - label: "I checked, but didn't find any duplicates ([open](https://github.com/TeamNewPipe/NewPipe/issues) OR [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed)) of this issue in the repo"
+          required: true
+        - label: "I have read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md)"
+          required: true
+        - label: "This issue contains only one feature request. I will open one issue for every feature I want to request."
```

Comment:
I'd rather avoid a checkbox for this, but put it as a description next to the form that asks for information on the issue. I would instead add a checkbox informing the user that when the form is not filled out correctly, their issue *will be closed*.

For example:

> I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

Reply:
> I'd rather avoid a checkbox for this

Having to disagree here.
We already got some issues that contained multiple requests / bugs.
If it's not stated here we have no guideline noted anywhere why we close the issue.

However the example below is pretty good.
I will removing the "However note the following before opening an issue:"-block in favor of it.